packages/server-core/src/social/location/location.resolvers.ts

```diff
@@ -137,7 +140,15 @@ export const locationPatchResolver = resolve<LocationType, HookContext>({
   updatedBy: async (_, __, context) => {
     return context.params?.user?.id || null
   },
+  maxUsersPerInstance: resolvedMaxUserPerInstance,
   updatedAt: getDateTimeSql
 })
 
+async function resolvedMaxUserPerInstance(value, location) {
+  if (location.maxUsersPerInstance > MAX_USER_PER_INSTANCE) {
+    throw new BadRequest('You have entered higher than the allowed max users. Please enter a number between 1 and 10.')
```

Comment:
`MAX_USER_PER_INSTANCE` is 100 but we are adding a range upto 10

Reply:
yea, I confirmed with ross yesterday. I changed it to 100 because test was failing. Ill push up a new commit to fix